app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+validates_presence_of :first_name, :last_name, :google_token, :self_care_time
+validates :email, uniqueness: true
+
```

Comment:
Great work here with starter User info. May need to adjust User attributes later depending on Oauth. We may not need to store many of these attributes.

Reply:
Thanks. I actually have only done Oauth in projects where you had the option to log in through the site as well so I wasn't aware of that.